src/drt/src/pa/FlexPA_prep.cpp

```diff
@@ -2700,10 +2701,10 @@ bool FlexPA::genPatterns_commit(
       }
     }
 
-    frAccessPoint* leftAP = nullptr;
```

Comment:
ap is fine by the way. it abbreviates accesspoint and is pretty standard so it's understandable.